aten/src/ATen/ATen.h

```diff
@@ -1,9 +1,5 @@
 #pragma once
 
-#if !defined(_MSC_VER) && __cplusplus < 201402L
```

Comment:
This error message is still useful and should not be removed. Some subprojects that extend pytorch may still encounter this error (especially since it is in a header).

Reply:
Instead of being removed, this should be raised to C++17 when all downstream libs are switched over. (That is the main constraint keeping us from having C++17 code in headers at the moment).

Reply:
I see